client/checkout/blocks/hooks.js

```diff
@@ -37,6 +37,28 @@ export const usePaymentCompleteHandler = (
 	);
 };
 
+/**
+ * Handles onCheckoutFail event emitter which fires after Blocks checkout processor responds with error.
+ *
+ * Displays the error message returned from checkout processor in the noticeContexts.PAYMENTS area.
+ *
+ * @param {*} onCheckoutFail The onCheckoutFail event.
+ * @param {*} emitResponse   Various helpers for usage with observer.
```

Comment:
nit: I think this could be more descriptive with something like:

```suggestion
 * @param {Function} onCheckoutFail The onCheckoutFail event emitter.
 * @param {Object} emitResponse   Various helpers for usage with observer.
```